deid/dicom/fields.py

```diff
@@ -86,11 +86,22 @@ def value_contains(self, expression):
                 return True
         return False
 
-    def tag_within_group(self, group):
+    def select_matches(self, expression):
         """
-        Determine whether the element tag belongs to a group
+        Determine whether the element has a specific selected attribute
         """
-        return self.element.tag.group == group
+        attribute, value = expression.split(":", 1)
```

Comment:
Yes! This is exactly what I had in mind - it matches the pattern of the other selectors because it's a generic term, e.g., func, startswith, etc.